modules/doc/content/application_development/profiling.md

```diff
@@ -268,7 +268,30 @@ Follow the steps below to get profiling information for your application:
 - Compile MOOSE and your application in `oprof` mode.
 - Run your application through the profiler:
 
-### Newer versions of MacOS (Mojave):
+### MacOS (Big Sur / Monterey / Ventura) with xCode 13 (and later):
+
+```
+xctrace record --template 'Time Profiler' --target-stdout - --launch -- mooseapp-oprof -i input.i
+```
+
+This will create a directory `Launch_mooseapp-oprof[0-9]+.trace`, where `[0-9]+`
+denotes a number, which you can open using
+
+```
+open Launch_mooseapp-oprof[0-9]+.trace
+```
+
+The Instruments application will open in a new window with the profile.
+The performance profile can be converted to [pprof](https://github.com/google/pprof)
+using [InstrumentsToPprof](https://github.com/google/instrumentsToPprof). To do this, open the trace in Instruments
+and select the processes you want to be converted, then copy the data using _Deep Copy_ in the _Edit_ menu. The copied
```

Comment:
```suggestion
and select the processes you want to be converted, then copy the data using *Deep Copy* in the *Edit* menu. The copied
```

"MooseDown" doesn't recognize the underscores for italics, so let's switch these to asterisks.